scripts/set-performance

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Please rename for consistency to something like `set-performance-settings` or `set-performance-cluster-settings`